docs/source/basics/defining_environment.md

```diff
@@ -4,7 +4,7 @@ Workflow code is rarely free of dependencies. It may require python or system pa
 
 Latch manages the execution environment of a workflow using Dockerfiles. A Dockerfile specifies the environment in which the workflow runs.
 
-Latch has three [base images](https://docs.docker.com/build/building/base-images/), one without hardware acceleration drivers, one with CUDA drivers, and one with OPENCL drivers. To use the CUDA or OPENCL base image, use the `--cuda` or `--opencl` flags when running `latch init`.
+Latch has four [base images](https://docs.docker.com/build/building/base-images/), one bare minimum, one with CUDA drivers, one with OPENCL drivers, and one with the Docker software pre-installed. To use the CUDA, OPENCL, or Docker base image, use the `--base-image` flag when running `latch init`.
```

Comment:
- "one bare minimum" is weird
- "OpenCL" not "OPENCL"
- "Docker software" is weird
- using a list can be simpler here
- don't repeat yourself in the second sentence
```suggestion
Each environment is build on one of the following base distributions:
- `default` with no additional dependencies
- `cuda` with Nvidia CUDA/cuDNN drivers !!!!!_ADD_VERSION_HERE_!!!!!
- `opencl` with OpenCL !!!!!_ADD_VERSION_HERE_!!!!! drivers
- `docker` with the Docker daemon

The base environment can be specified at workflow creation using [the `--base-image` flag of `latch init`](LINK TO THE SECTION IN LATCH INIT DOCS)
```